CHANGELOG.rst

```diff
@@ -36,6 +36,9 @@ Added
 
 - [debops.nginx] Support to disable logging per Nginx server.
 
+- [debops.docker] New role for Docker client management. The old debops.docker
+  role has been renamed to :ref:`debops.docker_server`.
+
```

Comment:
Usually new DebOps roles have their own subsection in the "Added" section to combine them in one place. You could add it if it isn't in the Changelog yet, see previous releases for how it was written.